Documentation: Document @storybook/test package (#24263)
storybookjs/storybook: docs/essentials/actions.md

Actions work via supplying special Storybook-generated “action” arguments (referred to as "args" for short) to your stories. There are two ways to get an action arg:

### Via @storybook/test fn spy function

> The first and recommended way to write actions, is to use the `fn` function from `@storybook/test`. This package provides a utility to help mock and spy args, which is very useful for writing tests with the [play function](../writing-stories/play-function.md). You can mock your component's methods by assigning them with the `fn()` function:

Reviewer: ```suggestion
The recommended way to write actions is to use the `fn` utility from `@storybook/test` to mock and spy args. This is very useful for writing [interaction tests](../writing-tests/interaction-testing.md). You can mock your component's methods by assigning them to the `fn()` function:
```

<!-- prettier-ignore-start -->

<CodeSnippets
  paths={[
    'angular/button-story-onclick-action-spy.ts.mdx',
    'common/button-story-onclick-action-spy.js.mdx',
    'common/button-story-onclick-action-spy.ts.mdx',
  ]}
/>

<!-- prettier-ignore-end -->

> If your component calls an arg which is spied on (based on the user's interaction or through the `play` function), the event will show up in the action panel:

Reviewer: ```suggestion
If your component calls an arg (because of either the user's interaction or the `play` function) and that arg is spied on , the event will show up in the action panel:
```
